docs/en/week08/08-3.md

```diff
@@ -199,7 +199,7 @@ Now let's have compare the difference between VAE and DAE. See *Fig. 6* below.
 <b>Fig. 6</b>: VAE vs. Denoising AE
 </center>
 
-For DAE as we mentioned before, the sampling was happening between <span style="color:#80b1d3">$\boldsymbol{y}$</span>, in blue, because it’s cold / low energy, and <span style="color:#fb8072">$\boldsymbol{\hat{y}}$</span>, in red, because it's hot / high energy. So we move the input and then we decode everything into $\boldsymbol{\tilde{y}}$. For VAE we encode the input and we add noise in the hidden , so we just switch the position of the encoder in the sampler more or less.
+For DAE as we mentioned before, the sampling was happening between <span style="color:#80b1d3">$\vy$</span>, in blue, because it’s cold / low energy, and <span style="color:#fb8072">$\vyhat$</span>, in red, because it's hot / high energy. So we move the input and then we decode everything into $\vytilde$. For VAE we encode the input and we add noise in the hidden , so we just switch the position of the encoder in the sampler more or less.
```

Comment:
```suggestion
For DAE as we mentioned before, the sampling was happening between $\vy$, in blue, because it’s cold / low energy, and $\vyhat$, in red, because it's hot / high energy. So we move the input and then we decode everything into $\vytilde$. For VAE we encode the input and we add noise in the hidden , so we just switch the position of the encoder in the sampler more or less.
```